esx_service/vmdk_ops.py

```diff
@@ -252,31 +253,64 @@ def getVMDKUuid(vmdk_path):
     except:
         return None
 
+# Create and return the devFS path for a VSAN object UUID.
+# Also returns true if clean up is needed.
+# cleanup_vsan_devfs should be called to clean up before
+# removing the VSAN object.
+def get_vsan_devfs_path(uuid):
+
+    logging.debug("Got volume UUID %s", uuid)
+
+    # Objtool creates a link thats usable to 
+    # read write to vsan object.
+    cmd = "{0} {1}".format(OBJ_TOOL_CMD, uuid)
+    rc, out = RunCommand(cmd)
+    fpath="/vmfs/devices/vsan/{0}".format(uuid)
+    if rc == 0 and os.path.isfile(fpath):
+        return fpath, True
```

Comment:
Nit: Isn't True always returned if fpath isn't None? Seems like this variable isn't needed at all.